src/ui/CategoryFacet/CategoryFacet.ts

```diff
@@ -56,6 +56,7 @@ export interface ICategoryFacetOptions extends IResponsiveComponentOptions {
   maximumDepth?: number;
   valueCaption?: IStringMap<string>;
   dependsOn?: string;
+  includeInBreadcrumb?: boolean;
```

Comment:
What is the use-case for someone configuring this option to `false`?

Reply:
Not wanting the values of that facet to appear in the breadcrumb, was an option in other DynamicFacet so I added it here